config/dev/dgutman-dsa-server-preprod-ec2.yaml

```diff
@@ -0,0 +1,27 @@
+# This stack is based on `config/prod/dgutman-htan-new2-ec2.yaml`
+# in https://github.com/Sage-Bionetworks/sandbox-provisioner/
+template_path: "remote/managed-ec2-linux-v2.j2"
+stack_name: "dgutman-dsa-server-preprod-ec2"
+
+sceptre_user_data:
+  Distro: "ubuntu"
+  OpenPorts: ["80", "443", "22"]
+parameters:
+  Department: "CompOnc"
+  Project: "htan"
+  OwnerEmail: "bruno.grande@sagebase.org"
+  InstanceType: "i3en.2xlarge"
+  AMIId: "ami-0b7906ab614596e7e"  # https://github.com/Sage-Bionetworks-IT/packer-base-ubuntu-bionic/releases/tag/v1.0.9
+  VolumeSize: "1000"
+  KeyName: "EmoryHealthcare-dgutman"  # Imported via AWS Management Console
+  VpcName: "htan-vpc"
```

Comment:
vpc is setup in a specific say on sandbox and scicomp, i'm not sure if it's compatible with this VPC setup.

Reply:
@zaro0508 what are the VPC parameters compatible w/ sandbox?

Reply:
@zaro0508: Where is this documented? I thought templates in `aws-infra` were general-purpose. Could you clarify what exactly you expect to cause issues?

Reply:
it's not necessarily `general purpose`, it's any template that you want to share and deploy to multiple accounts.  those files were used in the two accounts (sandbox & scicomp) that were setup for git based provisioning.  that template contains a lot more than you need for your purpose that's why i suggest you use your own template or just pair that template down into something much simpler.   One thing that will break is referenced to essentials which you do not have in your account. https://github.com/Sage-Bionetworks/aws-infra/blob/v0.2.13/templates/managed-ec2-ubuntu-v2.j2#L241

Reply:
I've added a copy of the `managed-ec2-linux-v2.j2` template to this repo directly on `main`. This way, any edits that I made to the template in this branch will come up as diffs. Note that I copied the instance role from `essentials.yaml`.